lib/Onyx.js

```diff
@@ -908,67 +921,43 @@ function multiSet(data) {
         .catch(error => evictStorageAndRetry(error, multiSet, data));
 }
 
-// Key/value store of Onyx key and arrays of values to merge
-const mergeQueue = {};
-
 /**
- * @private
- * @param {String} key
- * @returns {Boolean}
- */
-function hasPendingMergeForKey(key) {
-    return Boolean(mergeQueue[key]);
-}
```

Comment:
Why are we removing this functionality?

Reply:
oh, it just got moved down [here](https://github.com/chrispader/react-native-onyx/blob/d0f261ed20f4dd6295c870404a8edcfe46e34326/lib/Onyx.js#L1009-L1016)

I can move it back, just though it would fit more where the other merge-related functions are

Reply:
Never mind it seems it was moved further down